_data/projects/fresh_script.yml

```diff
@@ -0,0 +1,13 @@
+name: Fresh Script
+desc: Find Spotify tracks posted to the HipHopHeads subreddit and add them to a Spotify playlist. 
+site: https://github.com/amcquade/fresh_script
+tags:
+- python
+- reddit
+- spotify
+- music
+- rap
+- script
+upforgrabs:
+  name: fresh_script
```

Comment:
@amcquade Your label name here is "fresh_script" and it is "up-for-grabs" in the link followed. Please correct this.

Reply:
changed name to "up-for-grabs"